Makefile

```diff
@@ -39,7 +39,7 @@ build:
 	goreleaser build --clean --snapshot --single-target=${SINGLE_TARGET}
 
 release:
-	./bin/goreleaser release --clean
+	goreleaser release --clean
```

Comment:
I also think this won't be a big deal, but what problem were you solving here?  I think either way it would be picking goreleaser from the local `./bin` dir, because of the exported `$PATH` (line 8).  Was there another issue?

Reply:
Really, my goal here was to normalize how we called it and I picked the way `make build` did it, but either make the same sense to me.

Reply:
Works for me.  I was just curious about the change.